tests/integration-tests/tests/storage/test_efs.py

```diff
@@ -68,6 +73,129 @@ def test_efs_same_az(region, scheduler, pcluster_config_reader, clusters_factory
+
+    logging.info("cat hello_worlding")
+    file_name = existing_efs.random_file_name
+    remote_command_executor.run_remote_command(f"cat {mount_dir}/{file_name}")
+    scheduler_commands = get_scheduler_commands(scheduler, remote_command_executor)
+    _test_efs_correctly_mounted(remote_command_executor, mount_dir)
+    _test_efs_correctly_shared(remote_command_executor, mount_dir, scheduler_commands)
+    remote_command_executor.run_remote_command(f"cat {mount_dir}/{file_name}")
+
+
+@pytest.fixture()
+def existing_efs(vpc_stack, cfn_stacks_factory, request, region):
+    # efs stack contains a single efs resource
+    efs_template = Template()
+    efs_template.set_version("2010-09-09")
+    efs_template.set_description("EFS stack created for testing existing EFS")
+    efs_template.add_resource(FileSystem("FileSystemResource"))
+    efs_stack = CfnStack(
+        name="integ-tests-efs-{0}{1}{2}".format(
+            random_alphanumeric(),
+            "-" if request.config.getoption("stackname_suffix") else "",
+            request.config.getoption("stackname_suffix"),
+        ),
+        region=region,
+        template=efs_template.to_json(),
+    )
+    cfn_stacks_factory.create_stack(efs_stack)
+
+    # write file stack contains a mount target and a instance to write a empty file with random name into the efs
+    write_file_template = Template()
+    write_file_template.set_version("2010-09-09")
+    write_file_template.set_description("Stack to write a file to the existing EFS")
+    default_security_group_id = (
+        boto3.client("ec2", region_name=region)
+        .describe_security_groups(
+            Filters=[
+                {"Name": "vpc-id", "Values": [vpc_stack.cfn_outputs["VpcId"]]},
+                {"Name": "group-name", "Values": ["default"]},
+            ]
+        )
+        .get("SecurityGroups")[0]
+        .get("GroupId")
+    )
+    write_file_template.add_resource(
+        MountTarget(
+            "MountTargetResource",
+            FileSystemId=efs_stack.cfn_resources["FileSystemResource"],
+            SubnetId=vpc_stack.cfn_outputs["PublicSubnetId"],
+            SecurityGroups=[default_security_group_id],
+        )
+    )
+    efs_stack.random_file_name = random_alphanumeric()
+    user_data = (
+        """
+    #cloud-config
+    package_update: true
+    package_upgrade: true
+    runcmd:
+    - yum install -y amazon-efs-utils
+    - apt-get -y install amazon-efs-utils
+    - yum install -y nfs-utils
+    - apt-get -y install nfs-common
+    - file_system_id_1="""
+        + efs_stack.cfn_resources["FileSystemResource"]
+        + """
+    - efs_mount_point_1=/mnt/efs/fs1
+    - mkdir -p "${efs_mount_point_1}"
+    - mount -t efs ${file_system_id_1}:/ ${efs_mount_point_1}
+    - touch ${efs_mount_point_1}/"""
+        + efs_stack.random_file_name
+    )
+    write_file_template.add_resource(
+        Instance(
+            "InstanceToWriteEFS",
+            ImageId="ami-0c2f25c1f66a1ff4d",
```

Comment:
I think it is a alinux2 ami. If we test alinux+awsbatch, it is better to keep consistency.

Reply:
Good catch. I shouldn't hardcode the `ImageId`. But I still prefer using `alinux2` , instead of choosing the `OS` according to the test. 
There are 2 reasons for doing so:
1. I am not sure if the userdata commands work for any `OS`
2. The primary purpose of this fixture is to create a efs with a file inside. The `OS` of the instance to write the file does not impair the validity of the purpose